Makefile

```diff
@@ -62,6 +62,13 @@ release:
 image-minikube:
 	eval $$(minikube docker-env) && make image
 
+shellcheck:
+	docker run --rm --privileged \
```

Comment:
does it have to be priviliged?

Reply:
No, I just kept it as privileged by looking other commands in Makefile. I will remove it.